cmd/traffic_layout/engine.cc

```diff
@@ -127,6 +129,8 @@ help_message(const int versionflag, const int runflag, const int cleanflag, cons
     std::cout << "\nremove example: traffic_layout --remove(-r) /path/to/sandbox" << std::endl;
   if (forceflag)
     std::cout << "\nforce example: traffic_layout --force init /path/to/sandbox" << std::endl;
+  if (relocate_flag)
+    std::cout << "\ninit example: traffic_layout --init(-i) /path/to/sandbox --relocate" << std::endl;
```

Comment:
I think the examples wold be more helpful if you explained what they were doing. Maybe defer all the examples to the man page rather than the code?

Reply:
I am ok with simple usage examples.. people are not going to want to look in documentation for simple stuff